charts/kube-state-metrics/templates/crs-configmap.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if .Values.customResourceState.enabled}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "kube-state-metrics.fullname" . }}-customresourcestate-config
+data:
+  config.yaml: |
+        {{ .Values.customResourceState.config | toYaml | quote }}
```

Comment:
@dotdc Still not sure what the best way is to sanitize user input here and create a valid yaml file.